src/main/scala/com/fulcrumgenomics/commons/CommonsDef.scala

```diff
@@ -374,6 +372,13 @@ trait CommonsDef extends Compat {
     }
   }
 
+  /** Implicit class that allows generation of ParIterators from Iterators with convenience methods. */
+  implicit class ParIteratorSupport[A](iterator: Iterator[A]) {
+    def parWith(threads: Int, chunkSize: Int = 2048, chunkBuffer: Int = 2): ParIterator[A] = {
```

Comment:
I thought about adding `parWith(pool)` and `parWith(support)` here too, but it's complicated by the fact that we can't have multiple methods with the same name and default parameter values.  Since we hardly ever use those variants I've left them off for now.  We could solve this be either:

1. Not having defaults on the other methods
2. Giving them different names (e.g. parWithPool(...))
3. Having multiple implicit classes all with a single `parWith` method each

Reply:
Done.